app/controllers/membership_applications_controller.rb

```diff
@@ -49,15 +49,16 @@ def create
     if @membership_application.save
 
       if new_file_uploaded(params)
+
+        MembershipApplicationMailer.acknowledge_received(@membership_application).deliver
```

Comment:
It seems `deliver` is deprecated (?), so you might want to change this to `deliver_now`